contracts/.env.example

```diff
@@ -1,33 +1,12 @@
-### Salt used to deploy the contracts. Recommended to use the same salt across different chains.
-CREATE2_SALT=
-
-### The chains to deploy to, specified by chain name (e.g. CHAINS=mainnet,sepolia,arbitrum_sepolia)
-CHAINS=
-
-### RPCs for each chain ID
-RPC_MAINNET=
-RPC_BASE_SEPOLIA=
-RPC_ARBITRUM_SEPOLIA=
-RPC_SEPOLIA=
-
-### Etherscan API keys for each chain ID
+#### General variables
+PRIVATE_KEY=
+RPC_URL=
 ETHERSCAN_API_KEY_MAINNET=
-ETHERSCAN_API_KEY_BASE_SEPOLIA=
-ETHERSCAN_API_KEY_ARBITRUM_SEPOLIA=
-ETHERSCAN_API_KEY_SEPOLIA=
-
-### Etherscan API URLs for each chain ID
-ETHERSCAN_API_URL_MAINNET=https://api.etherscan.io/api
-ETHERSCAN_API_URL_HOLESKY=https://api-holesky.etherscan.io/api
-ETHERSCAN_API_URL_SEPOLIA=https://api-sepolia.etherscan.io/api
```

Comment:
Leave the Sepolia one?

Reply:
we dont do multi contract deployment on different chains